### DISCUSSION TITLE
Is there a rate limit for how quickly subfinder can be called?

### DISCUSSION THREAD

**Discussion by jheadings** [projectdiscovery/subfinder#561] [Q&A] [answered]
- If subfinder was used to repeated discover subdomains on a large number of domains would there be a limit to how quickly each domain can be queried?

**Comment by olearycrew** [comment, marked as answer]
Subfinder itself is a passive tool, so the only limit would be on those sources your specify, not on the identified targets. Rate limiting will come into play as you run HTTPx or Nuclei against those domains.
